app/src/main/java/org/schabi/newpipe/player/MainVideoPlayer.java

```diff
@@ -424,6 +440,20 @@ public void initViews(View rootView) {
             titleTextView.setSelected(true);
             channelTextView.setSelected(true);
 
+            getRootView().addOnLayoutChangeListener((view, l, t, r, b, ol, ot, or, ob) -> {
+                // Use smaller value to be consistent between screen orientations
```

Comment:
Little thing that's bothering the micro-optimization part of me, we could return if the size itself didn't changed:
```java
if (left == oldLeft && right == oldRight) return;
```

Reply:
What about `if (l != ol || t != ot || r != or || b != ob) { // do things... }` ?

Reply:
It's fine, this piece of code is short enough to be a good a choice.